library/src/test/java/ConflictStringToConflictMapConverterTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ConflictStringToConflictMapConverterTest {
+  ConflictStringToConflictMapConverter conflictStringToConflictMapConverter;
+
+  @BeforeEach
+  void init() {
+    conflictStringToConflictMapConverter = new ConflictStringToConflictMapConverter();
+  }
+
+  @Test
+  void testConvertEmptyConflictResolutionsString() throws IOException {
```

Comment:
Another way to format test names is something like:  
functionName_condition_outcome

convertConflictResolutionsStringToConflictMap_withEmptyConflictResolutions_suceeds